### ISSUE TITLE
Not working with angular 5

### ISSUE THREAD

**Issue by ArtworkAD** [MurhafSousli/ngx-progressbar#118]
### Bug Report or Feature Request (mark with an `x`)
```
- [x] bug report -> please search issues before submitting
- [ ] feature request
- [ ] question
```

### OS and Version?

OSX Yosemite

### Versions

angular/cli: 1.6.3
Angular v5.0

### Repro steps

```
ng new my-app
cd my-app
npm install --save @ngx-progressbar/core
```

add `NgProgressModule.forRoot()` to imports of `app.module`.

add

```
<ng-progress #progressBar></ng-progress>
<button (click)="progressBar.start()">Start</button>
<button (click)="progressBar.complete()">Complete</button>
```

to app.component.html

### The log given by the failure

ERROR TypeError: Cannot read property 'start' of undefined

### Desired functionality

Should display loading bar.

**Comment by hbliang** [comment]
+1

### Version
Angular CLI: 1.6.4
Node: 9.4.0
OS: darwin x64
Angular: 5.2.0

**Comment by MurhafSousli** [comment]
@ArtworkAD @hbliang just set any input, e.g. color

```html
<ng-progress color="green"></ng-progress>
```
[stackblitz](https://stackblitz.com/edit/angular-g9bbvk?file=app%2Fapp.component.ts)

Will fix this next release

**Comment by hbliang** [comment]
@MurhafSousli Awesome! Thx

**Comment by MurhafSousli** [comment]
welcome, this is fixed in v4.0.1

**Comment by vishwapratap** [comment]
yet not working for me also.
   "@ngx-progressbar/core": "^5.3.2",
 "@angular/cli": "~7.1.4",
@artjomzab  i follow your step can you help me.
